src/components/side_panel/chart/building_blocks/label_range/label_range.ts

```diff
@@ -5,19 +5,22 @@ import { SelectionInput } from "../../../../selection_input/selection_input";
 import { Checkbox } from "../../../components/checkbox/checkbox";
 import { Section } from "../../../components/section/section";
 
+export interface Option {
```

Comment:
Functional note (to discuss with PO): now when hovering a pie chart item, we don't show the dataset name at all. That can be a problem if we have a pie with multiple datasets, no ? (see pie chart in demo, I have no idea if the outer circle is the first or second dataset)

Reply:
Did he think about multi-dataset pie chart ? 😅 But maybe we do'nt care

Reply:
At first he didn't know we could make multi-dataset pie chart, and after I told him he said to keep the title hidden anyway :D